settings.h

```diff
@@ -71,6 +71,9 @@ bool IS_METRIC = true;
 // Change for 12 Hour/ 24 hour style clock
 bool IS_STYLE_12HR = false;
 
+// Change for HH:MM/ HH:MM:SS format clock
+bool IS_STYLE_HHMM = true; // false => HH:MM:SS
```

Comment:
This should be `false` by default as you would otherwise introduce a breaking change.

Reply:
Done !